packages/bpk-component-image/src/BpkBackgroundImage-test.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { colors } from 'bpk-tokens/tokens/base.es6';
+import { BpkBackgroundImage } from '../index';
```

Comment:
Does barely matter, but better import it directly from `./BpkBackgroundImage`. The test file is colocated, and in other cases we test things that aren't publicly exposed in `index.js`.